pkg/mounter/safe_mounter_windows.go

```diff
@@ -411,7 +412,7 @@ func newCSIProxyMounter() (*csiProxyMounter, error) {
 	}, nil
 }
 
-func NewSafeMounter(enableWindowsHostProcess, useCSIProxyGAInterface bool) (*mount.SafeFormatAndMount, error) {
+func NewSafeMounter(enableWindowsHostProcess, useCSIProxyGAInterface bool, maxConcurrentFormat int, concurrentFormatTimeout time.Duration) (*mount.SafeFormatAndMount, error) {
 	if enableWindowsHostProcess {
 		klog.V(2).Infof("using windows host process mounter")
 		return &mount.SafeFormatAndMount{
```

Comment:
pls also make the same changes on windows host process

Reply:
Should we also use it in csiproxyGA nad csiproxyv1beta?

Reply:
not necessary since we are using windows host process deployment only now.